### PR TITLE
Updated versions to Calico 2.0 and latest Flannel

### PR DESCRIPTION
**Pull request by heschlie**

### CODE REVIEW

**Comment by caseydavenport** [comment]
@heschlie you should update to v2.0.1 :)

**Comment by heschlie** [comment]
@casey, and done!

**Comment by caseydavenport** [comment]
Could you also update all the images to use the `quay.io/` variants for consistency?

File: k8s-install/kubeadm/canal.yaml [hunk 1]
```diff
@@ -130,7 +130,7 @@ spec:
         # Runs the flannel daemon to enable vxlan networking between
```
Comment (caseydavenport) [inline k8s-install/kubeadm/canal.yaml]: Still missing one image:

```
          image: calico/cni:v1.5.5
```

**Comment by caseydavenport** [comment]
Ok, this LGTM

**Pull request merged**